package.json

```diff
@@ -7,7 +7,7 @@
     "prestart": "./git-version",
     "start": "ng serve",
     "prebuild": "./git-version",
-    "build": "ng build --prod --build-optimizer",
+    "build": "ng build --prod --build-optimizer --vendor-chunk=true",
```

Comment:
https://github.com/angular/angular-cli/wiki/build#--build-optimizer-and---vendor-chunk

This will be removed, once we are all set with plotly bundle size issue.